src/messages.js

```diff
@@ -421,4 +423,58 @@ export const sayDiamondAlready = (isModerator, wasModerator) => {
 
     const [, message] = messageMap.find(([condition]) => condition) || [];
     return message || `diamonds are forever!`;
+};
+
+/**
+ * @summary builds a "number of positions" message
+ * @param {BotConfig} _config bot configuration
+ * @param {Election} election current election
+ * @param {string} _text message content
+ * @returns {string}
+ */
+export const sayNumberOfPositions = (_config, election, _text) => {
+    const { numPositions = 0 } = election;
+
+    const suffix = pluralize(numPositions, "s", "");
+    const pastBe = pluralize(numPositions, "were", "was");
+    const currBe = pluralize(numPositions, "are", "is");
+    const future = pluralize(numPositions, "will be", "shall be");
+
+    const rules = [
+        [election.isActive(), currBe],
+        [election.isNotStartedYet(), future],
+        [election.isEnded(), pastBe]
+    ];
+
+    const [_rule, modal] = rules.find(([rule]) => rule) || [, `${currBe}/${pastBe}/${future}`];
+
+    return `${numPositions} mod${suffix} ${modal} elected`;
+};
+
+/**
+ * @param {BotConfig} config bot configuration
+ * @param {Election} election current election
+ * @param {string} text message content
+ * @returns {Promise<string>}
+ */
+export const sayUserEligibility = async (config, election, text) => {
+    const userId = +text.replace(/\D/g, "");
+
+    const { chatDomain, apiKeyPool } = config;
+
+    const apiSlug = chatDomain.split(".").slice(0, -1).join(".");
+
+    const userBadges = await getBadges(config, userId, apiSlug, getStackApiKey(apiKeyPool));
+
+    const requestedUser = await getUserInfo(config, userId, apiSlug, getStackApiKey(apiKeyPool));
+
+    if (!requestedUser) {
+        return `Can't answer now, please ask me about it later`;
+    };
+
+    const { isEligible } = calculateScore(requestedUser, userBadges, election);
```

Comment:
Stack overflow has required badges too, and I think Ubuntu has another prerequisite of signing some agreement that we can't verify.

Reply:
Yeah, it is more of a command to help out with checking if score calc works as expected - I propose not documenting it for now too, just keeping it around. Can be moved to a dev command just to be extra sure too. 

Re: Ubuntu - I've seen the path in your code that mentions candidates need to sign the Ubuntu Code of Conduct but never investigated it. As an easy option, the command can be extended with a check if we are on Ubuntu and a reply like "make sure you signed..." to be on the safe side

Reply:
If you can move it to dev-only for now, we can merge this PR

Reply:
Will do that tomorrow - have a couple of things to address today work-wise...